pkg/controller/security/security.go

```diff
@@ -0,0 +1,37 @@
+package security
+
+import corev1 "k8s.io/api/core/v1"
+
+var readOnlyRootFilesystem = false
+var allowPrivilegeEscalation = false
+var runAsNonRoot = true
+
+// See: https://github.com/operator-framework/operator-registry/blob/master/Dockerfile#L27
+var runAsUser int64 = 1001
+
+var containerSecurityContext = &corev1.SecurityContext{
```

Comment:
Do we need to specify the "privileged" field here too, or is that the default value and we don't need to be explicit here?

Reply:
privileged does indeed default to false. But I have made it explicit, just in case. If I'm reading the [docs](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container) correctly, the container should inherit the seccomp profile from the pod security context. So, I think we should be good. All good call outs!